src/content/preprocessed-profile-versioning.js

```diff
@@ -110,4 +110,12 @@ const _upgraders = {
       }
     }
   },
+  [3]: profile => {
+    // add debugPath: ""
```

Comment:
Could you add a few more words to be a bit more specific here. I think my future self would appreciate that.